.travis.yml

```diff
@@ -24,13 +24,15 @@ env:
   - REMOVE_XDEBUG="0"
   - REMOVE_XDEBUG="1"
 
-sudo: false
+sudo: required
 
 cache:
   directories:
     - $HOME/.composer/cache
 
 before_install:
+  - sudo apt-get update --yes
+  - sudo apt-get install --yes net-tools
```

Comment:
I think no. It\'s just a habit. Before `apt-get install` I need to do `apt-get update`